src/abstractdataframe/sort.jl

```diff
@@ -272,6 +272,8 @@ function Base.issorted(df::AbstractDataFrame, cols_new=[]; cols=[],
                      :issorted)
         cols_new = cols
     end
+    cols_new isa Union{Real, Symbol} && return issorted(df[cols_new], lt=lt, by=by, rev=rev, order=order)
```

Comment:
Given that there are three branches here, I'd find it clearer to use `if... elseif... else`.

Reply:
fixed